omero/developers/Web/CreateApp.txt

```diff
@@ -254,11 +254,32 @@ auto-generated by Django - see :doc:`/developers/Web/WebGateway`.
 Resources for writing your own code
 -----------------------------------
 
-The webtest app has a number of examples. If you go to the webtest
-homepage e.g. `<http://localhost:8000/webtest>`_ you will
+The `webtest <https://github.com/openmicroscopy/webtest/>`_ app
+has a number of examples. Once installed, you can go to the webtest
+homepage e.g. `<http://localhost:4080/webtest>`_ and you will
 see an introduction to some of them. This page tries to find random
 image and dataset from your OMERO server to use in the webtest examples.
 
+Using jQuery & jQuery UI from OMERO.web
```

Comment:
Can we get rid of this '&' please? You could use a '/' in this case I think rather than writing out 'and'